integrations/wallets/metamask.md

```diff
@@ -36,21 +40,27 @@ Next, navigate to the Networks tab and click on the "Add Network" button.
 
 Here you can configure MetaMask for the following networks:
 
-Moonbeam development node:
+=== "Development Node"
 
---8<-- 'text/metamask-local/development-node-details.md'
+    --8<-- 'text/metamask-local/development-node-details.md'
 
-Moonbase Alpha TestNet:
+=== "Moonbase Alpha TestNet"
 
---8<-- 'text/testnet/testnet-details.md'
+    --8<-- 'text/testnet/testnet-details.md'
+
+=== "Moonriver"
+
+    - Network Name: `Moonriver`
+    - New RPC URL: `{{ networks.moonriver.rpc_url }}`
+    - ChainID: `{{ networks.moonriver.chain_id }}`
+    - Symbol (Optional): `MOVR`
```

Comment:
I really like the table for this. Can we add block explorer info? Specially for Moonbase Alpha and Moonriver